CiscoISE Content (#24334)
demisto/content: Packs/cisco-ise/README.md

# Cisco ISE 
This pack includes Cortex XSIAM content. 
## Configuration on Server Side

> Follow the below steps to configure basic log syslog collection:

Reviewer: ```suggestion
Complete the following to configure basic log syslog collection:
```

1. Go to **Administration** > **System** > **Logging** > **Remote Logging Targets**
2. Click **Add** and then fill the required details.

> 3. Click **Save**, and then verify the creation of the new target by going to **Remote Logging Targets** page. 

Reviewer: ```suggestion
3. Click **Save**, and then verify the creation of the new target by going to the **Remote Logging Targets** page. 
```

> Note:

Reviewer: ```suggestion
**Note:**
```

> To prevent log segmentation, please set the Maximum Length of the log to **8096**.

Reviewer: ```suggestion
To prevent log segmentation, set the Maximum Length of the log to **8096**.
```
